app/[contentType]/[slug]/page.tsx

```diff
@@ -0,0 +1,189 @@
+import { Metadata } from 'next';
+import Image from 'next/image';
+import { notFound } from 'next/navigation';
+import { Chips, Container } from '../../../components';
```

Comment:
Changed the absolute imports to relative imports, because we use the Upleveled ESLint `tsconfig.json` and don't have paths defined
```
    "paths": {
      "@components": [
        "./components"
      ],
      "@utils/*": [
        "./utils/*"
      ],
    }
```